tdrs-backend/gunicorn_start.sh

```diff
@@ -7,4 +7,4 @@ python manage.py migrate
 python manage.py populate_stts
 python manage.py collectstatic --noinput
 echo "Starting Gunicorn"
-exec gunicorn tdpservice.wsgi:application --bind 0.0.0.0:8080 --timeout 10 --workers 3 --log-file=- --log-level $LOGGING_LEVEL
+exec gunicorn tdpservice.wsgi:application --bind 0.0.0.0:8080 --timeout 10 --workers 3 --reload --log-level $LOGGING_LEVEL
```

Comment:
The default value for `log-file` is `'-'` so this cfg is redundant. I have added the reload function so that when we make "live" dev changes they affect the application for better troubleshooting. 

As a TODO, we should *not* have this be true in production I imagine. Hopefully such a check will be simple in this gunicorn script.